adapter/internal/oasparser/envoyconf/envoyconf_internal_test.go

```diff
@@ -321,48 +311,6 @@ func TestGenerateRegex(t *testing.T) {
 			message:       "when provide a incorrect path",
 			isMatched:     false,
 		},
-		{
-			inputpath:     "/v2/pet/findById",
-			userInputPath: "/v2/pet/findById?status=availabe",
-			message:       "when query parameter is provided",
-			isMatched:     true,
-		},
-		{
-			inputpath:     "/v2/pet/findById",
-			userInputPath: "/v2/pet/findByIdstatus=availabe",
-			message:       "when query parameter is provided without ?",
-			isMatched:     false,
-		},
-		{
-			inputpath:     "/v2/pet/*",
-			userInputPath: "/v2/pet/findByIdstatus=availabe",
-			message:       "when the resource ends with *",
-			isMatched:     true,
-		},
-		{
-			inputpath:     "/v2/pet/*",
-			userInputPath: "/v2/pet/",
-			message:       "when the resource ends with *, empty string with / substitution fails.",
-			isMatched:     true,
-		},
-		{
-			inputpath:     "/v2/pet/*",
```

Comment:
Any reason for removing these unit tests?

Reply:
Will  check